src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/OrganisationInternalUserFunctionalTest.java

```diff
@@ -0,0 +1,54 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import lombok.extern.slf4j.Slf4j;
+import net.thucydides.core.annotations.WithTag;
+import net.thucydides.core.annotations.WithTags;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.hmcts.reform.lib.util.serenity5.SerenityTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.http.HttpStatus.NOT_FOUND;
+import static org.springframework.http.HttpStatus.NO_CONTENT;
+import static org.springframework.http.HttpStatus.OK;
+import static uk.gov.hmcts.reform.professionalapi.domain.OrganisationStatus.REVIEW;
+
+@SerenityTest
+@SpringBootTest
+@WithTags({@WithTag("testType:Functional")})
+@Slf4j
+@SuppressWarnings("unchecked")
+class OrganisationInternalUserFunctionalTest  extends AuthorizationFunctionalTest {
+
+    @Test
+    @DisplayName("PRD Internal Delete Organisation with status REVIEW Test Scenarios")
+    void testInternalOrganisationDeleteScenario() {
```

Comment:
move this test to ProfessionalInternalUserFunctionalTest

Reply:
As we are adding multiple test cases in same class ProfessionalInternalUserFunctionalTest , Separated it into separate class and in future any test cases related to organization should be added to OrganisationInternalUserFunctionalTest

Reply:
do you mean currently ProfessionalInternalUserFunctionalTest does not contain test cases related to organisation and so that's the reason you created new test class only for organisation related test cases ?

Reply:
It is containing integration related to organisation and other functionality. If we add more test cases it will not be maintainable in future so created new class and we can add organisation related test cases in new class.